app/helpers/application_helper/button/host_initiator_group_new.rb

```diff
@@ -11,10 +11,10 @@ def calculate_properties
   def disabled?
     if @view_context.params["id"]
       current_ems = ExtManagementSystem.where(:id => @view_context.params["id"]).first
```

Comment:
Unrelated but would be a good follow-up, this looks like it'd be better as `current_ems = ExtManagementSystem.find(@view_context.params["id"])`